pages/common/unimatrix.md

```diff
@@ -0,0 +1,25 @@
+# unimatrix
+
+> A script to simulate the Matrix look with Unicode characters.
+> See also: `cmatrix`.
+> More information: <https://github.com/will8211/unimatrix>.
+
+- Mimic the default output of `cmatrix` (no unicode, works in TTY):
+
+`unimatrix --no-bold --speed={{96}} --character-list={{o}}`
```

Comment:
```suggestion
- Mimic the default output of `cmatrix` (no unicode, works in a TTY):

`unimatrix --no-bold --speed {{96}} --character-list {{o}}`
```

Reply:
Does unimatrix not require the equal signs? The documentation only gives examples with equal signs, so I'm not sure about this.

Reply:
I tested it, `--opt arg` works exactly the same as `--opt=arg`.